views/js/e2e/_helpers/commands/setupCommands.js

```diff
@@ -52,24 +130,41 @@ Cypress.Commands.add('importTestPackage', (fileContent, fileName) => {
             force: true
         }
     );
-
     cy.wait('@fileUpload');
 
-    // Import selected example test file
+    // Import selected test file
     cy.get(setupSelectors.testsPage.fileImportButton).click();
 
-    // Wait until test import request finishes
-    cy.wait(['@testImportIndex', '@taskQueueWebApi', '@taskQueueWebApi'], { timeout: 15000 });
+    /*
+     * Two different processes to handle different UX
+     * for taoTaskQueue enabled or disabled
+     */
+    cy.get('@isTaskQueueWorking').then(working => {
+        if (working) {
+            cy.wait(Array(4).fill('@taskQueueWebApi'));
```

Comment:
5000 ms was not enough for my computer.

Reply:
I added a 20000ms timeout. If it becomes a widespread problem, we can move that value into the request defaults in `cypress.json`.